### PR TITLE
Add Ikara

### PR DESCRIPTION
**Pull request by oipika**
## Type of change

- [ ] Add new directory for a server
- [ ] Update directory of a server
- [ ] Remove directory of a server
- [ ] Documentation Update

## Checklist

- [ ] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [ ] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [ ] I have created the manifest.json with the required values

### CODE REVIEW

File: minecraft_servers/ikara/background.png
Comment (maaaathis) [inline minecraft_servers/ikara/background.png:1]: Compress this file

File: minecraft_servers/ikara/background@2x.png
Comment (maaaathis) [inline minecraft_servers/ikara/background@2x.png:1]: Compress this file

File: minecraft_servers/ikara/manifest.json [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  "server_wildcards": [
+    "%.ikara.gg"
+  ],
+  "supported_languages": [
+     "en"
+  ],
+  "social": {
+    "web": "https://ikara.gg",
+    "web_shop": "https://store.ikara.gg",
+    "web_support": "https://ikara.gg/discord",
```
Comment (maaaathis) [inline minecraft_servers/ikara/manifest.json:14]: `web_support` cannot be a discord server or forum; use a contact form, faq or similar or remove

File: minecraft_servers/ikara/manifest.json [hunk 2]
```diff
@@ -0,0 +1,50 @@
+{
+  "server_name": "ikara",
+  "nice_name": "Ikara",
+  "direct_ip": "ikara.gg",
+  "server_wildcards": [
+    "%.ikara.gg"
```
Comment (maaaathis) [inline minecraft_servers/ikara/manifest.json:6]: Doesn't look like a real wildcard
image

File: minecraft_servers/ikara/manifest.json [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+  "nice_name": "Ikara",
+  "direct_ip": "ikara.gg",
+  "server_wildcards": [
+    "%.ikara.gg"
+  ],
+  "supported_languages": [
+     "en"
+  ],
+  "social": {
+    "web": "https://ikara.gg",
```
Comment (maaaathis) [inline minecraft_servers/ikara/manifest.json:12]: No real content; if website does not exist, remove it

File: minecraft_servers/ikara/manifest.json [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+    "twitter": "PlayIKARA",
+    "tiktok": "PlayIkara",
+    "discord": "https://discord.gg/BzUyWTCWD2",
+    "youtube": "https://www.youtube.com/@Ikaranetwork"
+  },
+  "gamemodes": {
+    "factions": {
+      "name": "Factions", 
+      "color": "#7CFDFF",
+      "url": "https://www.Ikara.gg/Discord",
```
Comment (maaaathis) [inline minecraft_servers/ikara/manifest.json:24]: Reserved for a own landing page for your gamemode, not a general url

File: minecraft_servers/ikara/manifest.json [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+      "command": "/server factions"
+    },
+    "practice": {
+      "name": "Practice", 
+      "color": "#FFE287",
+      "url": "https://www.Ikara.gg/Discord",
```
Comment (maaaathis) [inline minecraft_servers/ikara/manifest.json:30]: Reserved for a own landing page for your gamemode, not a general url

File: minecraft_servers/ikara/manifest.json [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+      "command": "/server practice"
+    },
+    "survival": {
+      "name": "Survival",
+      "color": "#8CFCB9",
+      "url": "https://www.Ikara.gg/Discord",
```
Comment (maaaathis) [inline minecraft_servers/ikara/manifest.json:36]: Reserved for a own landing page for your gamemode, not a general url

**Review by maaaathis — CHANGES_REQUESTED** [review]

**Review by DoJapHD — APPROVED** [review]

**Pull request merged**